packages/hydrogen/src/cart/createCartHandler.test.ts

```diff
@@ -58,7 +58,7 @@ describe('createCartHandler', () => {
       },
     });
 
-    expectTypeOf(cart).toEqualTypeOf<HydrogenCartCustom<{}>>;
+    expectTypeOf(cart).toEqualTypeOf<HydrogenCartCustom<{foo: () => 'bar'}>>;
```

Comment:
@wizardlyhel The change in https://github.com/Shopify/hydrogen/pull/1588/files#r1430177086 is basically to fix this code here.

It was complaining about `HydrogenCartCustom<{}>` and it wouldn't allow us to pass `foo` as a custom method in this type check. With that change, we can now specify what we expect of the type, which I think it makes sense?